tests/test_browsing_context.py

```diff
@@ -1175,19 +1175,16 @@ async def test_browsingContext_create_withUserGesture_eventsEmitted(
         {"method": "browsingContext.domContentLoaded"})
     assert load_event == AnyExtending({"method": "browsingContext.load"})
 
-    session_id = await get_cdp_session_id(context_id)
-
-    # XXX: Execute via BiDi once supported: https://github.com/w3c/webdriver-bidi/issues/359
     command_id = await send_JSON_command(
         websocket, {
-            "method": "cdp.sendCommand",
+            "method": "script.evaluate",
```

Comment:
Also remove the now unused fixture `get_cdp_session_id` from the method (same below)